.husky/pre-commit

```diff
@@ -1,2 +1 @@
 bunx lint-staged
-prettier -c .
```

Comment:
Why did you remove the prettier here?

Reply:
Then should we remove husky? Or is it doing more?

Also, prettier seems to be pretty happy:

```
showcase_v2 [main] % prettier -c .
Checking formatting...
All matched files use Prettier code style!
```